LICENSE-APACHE

```diff
@@ -175,7 +175,7 @@
 
    END OF TERMS AND CONDITIONS
 
-   Copyright 2015-2018 Sean Griffin
+   Copyright 2015-2021 Sean Griffin
```

Comment:
We should likely update that to something like
```suggestion
   Copyright 2015-2018 Sean Griffin, 2018-2021 Diesel Core Team 
```
or something like that. 
cc @diesel-rs/core